facebook/delphiFacebook/R/responses.R

```diff
@@ -500,6 +507,28 @@ module_assignment <- function(input_data, wave) {
   return(input_data)
 }
 
+#' Label arms of experimental Wave 12.
+#' 
+#' @param input_data data frame of responses, before subsetting to select
+#'   variables
+#' @param wave integer indicating survey version
+#' 
+#' @return data frame with new `module` column
+#' @importFrom dplyr case_when
+experimental_arm_assignment <- function(input_data, wave) {
+  if (wave == 12.5) {
+    assert( "random_num_exp" %in% names(input_data) )
+    input_data$w12_treatment <- case_when(
+      input_data$random_num_exp >= 0.6666 ~ 1, # demographics placed after symptom items
+      input_data$random_num_exp >= 0.3333 ~ 2, # demographics placed after vaccine items
+      input_data$random_num_exp < 0.3333 ~ 3, # alternative wording to V1
```

Comment:
Have we verified with Adrianne that the Qualtrics setup will be using the same number of decimal places and the same choice of `>=` and `<`? Wouldn't want to have goofy issues where 0.1% of people are misclassified because of boundary cases

Reply:
Yup, this logic is taken directly from Adrianne's specifications.